src/main/webapp/one-off/databaseInit.java

```diff
@@ -40,104 +40,107 @@
 
 package com.google.sps.servlets;
```

Comment:
Please remove this file from the CL, it's gonna conflict with Paul's!

This is the prettifier running over all files.